index.html

```diff
@@ -0,0 +1,151 @@
+                    <span class="htcj2">Javascript</span><br><br>
+                      <div class="btn2">
+                        <a href="image2.png" target="_blank">See Project</a>
+                      </div> 
+                      </div>
+            </div>    
+           </div>
+           <div class="container">
+            <div class="imgdiv">
+                <img src="image3.png" alt="Nature" height="300" width="300">
+            </div>
+            <div class="tonic">
+                    <h1 class="popheading">Tonic</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
+                      <div class="btn3">
+                        <a href="image3.png" target="_blank">See Project</a>
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img src="image4.png" alt="Nature" height="220px" width="295px">
+            </div>
+                   <div class="multi">
+                    <h1 class="popheading">Multi Post Series</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
+                      <div class="btn4">
+                        <a href="image4.png" target="_blank">See Project</a>
+                      </div> 
+                      </div>
+                    
+            </div>    
+           </div>
+           <div class="container5">
+            <div class="about">
+                <h1 class="abouth">About Myself</h1>
+                <p class="popp15">Hello I’m a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what you see and
+                     have a project you need coded, don’t hestiate to contact me.</p>
+                 <span class="popp15">Let's Connect</span>   
```

Comment:
- [ ] The title of this text must use a header tag ie. (`h1,h2,h3 ..)` and not a span tag. This is because it is a subtitle of the section, not a span text :100:

Reply:
Suggested changes made